sdv/evaluation/single_table.py

```diff
@@ -43,12 +43,12 @@ def run_diagnostic(real_data, synthetic_data, metadata, verbose=True):
             Defaults to True.
 
     Returns:
-        dict:
-            The diagonstic results.
+        DiagnosticReport:
+            Single table diagnostic report object.
     """
     quality_report = DiagnosticReport()
     quality_report.generate(real_data, synthetic_data, metadata.to_dict(), verbose)
-    return quality_report.get_results()
+    return quality_report
```

Comment:
minor thing I'm noticing now but these variables should probably be named `diagnostic_report`